### PR TITLE
consistent css

### PR DESCRIPTION
**Pull request by the-j0k3r**
just a cleanup for consistency sake and my sensitive retinas, nothing new being added

### CODE REVIEW

**Comment by the-j0k3r** [comment]
@BrainSlayer after review curl file `https://github.com/mirror/dd-wrt/pull/52.patch` and patch apply wanna see if it keeps my info on it

cheers Im feeling better that css is finally up to my snuff =)

**Comment by the-j0k3r** [comment]
also added ftp site to trac theme

see

wut

https://github.com/style-it-themes/dd-wrt-trac-inspired-themes

The only thing left is the wiki, but Im not that way inclined. due to lack of feedback on existing stuff.

File: src/router/kromo/dd-wrt/elegant/fresh.css [hunk 1]
```diff
@@ -191,12 +188,11 @@ table[summary="ports forwarding table"] .num[size="5"] {
 table[summary="ports forwarding table"] input[size="10"] {
   max-width: 79px;
 }
-/* form select wds margin fix */
 form[name="wds"] select {
 	margin-right: 6px;
 }
 #header {
-  height: 9.240em;
+  height: 9.24em;
```
Comment (the-j0k3r) [inline src/router/kromo/dd-wrt/elegant/fresh.css:195]: extra 0 is not adding something, precision matters when the value is `< 1`

File: src/router/kromo/dd-wrt/elegant/fresh.css [hunk 2]
```diff
@@ -79,23 +80,24 @@ input {
...
   line-height: 1.42857;
   color: #555;
-  background-color: #FFF;
+  background-color: #fff;
   background-image: none;
-  border: 1px solid #CCC;
+  border: 1px solid #ccc;
   border-radius: 4px;
-  box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.075) inset;
-  transition: border-color 0.15s ease-in-out 0s, box-shadow 0.15s ease-in-out 0s;
```
Comment (the-j0k3r) [inline src/router/kromo/dd-wrt/elegant/fresh.css:87]: `0px` is only `0` unnecessary crud and also `0.075` you dont need it `.075` is enough, now if you remove the dot it matters.

none of these changes ADD bits to file size.

File: src/router/kromo/dd-wrt/elegant/fresh-dark.css [hunk 3]
```diff
@@ -162,13 +162,13 @@ table tr:not([bgcolor]):hover {
...
+  background-color: #383838;
 }
 td[bgcolor="#C0C0C0"]:hover {
-	background-color: #1a1a1a;
+  background-color: #1a1a1a;
 }
 form[name="wds"] select {
-	margin-right: 6px;
+  margin-right: 6px;
 }
```
Comment (the-j0k3r) [inline src/router/kromo/dd-wrt/elegant/fresh-dark.css:172]: indent fix only on these 3 rules  to conform to the indenting style, this is consistency only and not mixing tabs/spaces.

File: src/router/kromo/dd-wrt/elegant/fresh-dark.css [hunk 4]
```diff
@@ -137,7 +137,7 @@ select {
 select[disabled] {
   color: #666;
   background-color: #1a1a1a;
-  border-color: #111111;
+  border-color: #111;
 }
```
Comment (the-j0k3r) [inline src/router/kromo/dd-wrt/elegant/fresh-dark.css:141]: You dont need to write out 6 numbers in some cases, 3 is enough. I dont know how many bits its saves, but if you do this often enough it matters.

The browsers can read and understand this and besides, its inconsistent to mix short hex and long hex on same file and mix lowercase and  uppercase and if like me your eyes are perfect, its hard to read hex values where `8B` appear together, so lowercase helps me a great deal in general.

File: src/router/kromo/dd-wrt/wikar/style.css [hunk 5]
```diff
@@ -229,4 +229,4 @@ input.spaceradio, .submitFooter input.button {
     margin-bottom: 2px;
     width: 100px;
   }
-}
```
Comment (the-j0k3r) [inline src/router/kromo/dd-wrt/wikar/style.css:232]: if you read down to here, see how many unnecessary `px` units are being removed from `0` values? surely this is less NOT more file size? Again it is not necessary. it maybe was one million years ago.

It is relevant, Im removing unnecessary  crud, I dont see how removing unecessary stuff adds to ANY file size.

**Comment by BrainSlayer** [comment]
ahm you know that i already merged it?

**Pull request closed without merging**